### ISSUE TITLE
[BUG] Enable WCAG

### ISSUE THREAD

**Issue by jolevesq** [Canadian-Geospatial-Platform/geoview#2942]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

When we tab to the viewer and bring the WCAG box to enable and click enable, the map is in WCAG focus. Pressing tab again will remove the WCAG mode.

### Expected Behavior

WCAG mode should be enable after tab

### Steps To Reproduce

1. Go any pages
2. Tab to the map and bring the WCAG window
3. Click Enable (focus in on the map)
4. Tab again, you are kick out.

**Comment by jolevesq** [comment]
Cannot reproduce
